performance/README.md

```diff
@@ -0,0 +1,59 @@
+# Internal User Document: Load Test Performance Analysis Script
+
+## Introduction
+The `load_test_performance_analysis.py` script is designed to conduct load testing on SM-SDK APIs. Currently, it tests three APIs: `get_cycles`, `get_line_data`, and `get_kpi_data_viz` to analyze their performance metrics. This document provides a quick guide on how to use the script.
+
+## Prerequisites
+Before using the script, ensure the following prerequisites are met:
+
+- Python 3.8.18 and above installed on your system.
+- Necessary Python packages installed. You can install the required packages using pip:
+  ```
+  pip install pandas
```

Comment:
Are the performance tests pandas version dependent?

Reply:
No. It is not required.